### ISSUE TITLE
cells-sync command bg start not existing inside the application.

### ISSUE THREAD

**Issue by kirajesus** [pydio/cells-sync#47]
Hi fellas do you experience the same issue? In the documentation, bgstart is included but when you run the command ./cells-sync --help bgstart didn't show in the command list.

**Comment by kirajesus** [comment]
Hello, sir @cdujeu here is the documentation that I found. There is the bgstart command.

Other available commands
Use help to display the available commands:

$ ./cells-sync --help
Usage:
  ./cells-sync [flags]
  ./cells-sync [command]

Available Commands:
  add         Add a new task via command line
  autotest    Basic unidirectional sync between two local folders (under your temporary directory)
  bgstart     Start sync tasks from within service
  capture     Capture snapshots inside JSON file - do not perform any actual tasks
  delete      Delete existing sync via command line
  edit        Exit existing sync via command line
  help        Help about any command
  service     Manage service: install,uninstall,stop,start,restart
  start       Start sync tasks
  systray     Launch Systray
  version     Display version
  webview     Launch WebView

Flags:
  -h, --help   help for ./cells-sync

Use "./cells-sync [command] --help" for more information about a command.

**Comment by cdujeu** [comment]
ha, ok - yes, it's a Darwin only command used only when cells is controlled by OSX/Launchd.
Maybe you are looking for `start --headless` instead?

**Comment by kirajesus** [comment]
> ha, ok - yes, it's a Darwin only command used only when cells is controlled by OSX/Launchd. Maybe you are looking for `start --headless` instead?

Yup. maybe this is the command that I'm looking for. I'll try this. Thank you!
